website/source/docs/configuration/client.html.md

```diff
@@ -64,10 +64,14 @@ driver) but will be removed in a future release.
 - `network_interface` `(string: varied)` - Specifies the name of the interface
   to force network fingerprinting on. When run in dev mode, this defaults to the
   loopback interface. When not in dev mode, the interface attached to the
-  default route is used. All IP addresses except those scoped local for IPV6 on
-  the chosen interface are fingerprinted. The scheduler chooses from those IP
+  default route is used. The scheduler chooses from these fingerprinted IP 
   addresses when allocating ports for tasks.
 
+    Nomad will prefer non-local IP addresses.  If no non-local IP addresses are
+  found, Nomad will fingerprint link-local IPv6 addresses based on the 
```

Comment:
Suggested rewording: 
If no non-local IP addresses arefound, Nomad will fingerprint link-local IPv6 addresses if the [`"fingerprint.network.disallow_link_local"`](#quot-fingerprint-network-disallow_link_local-quot-) is false. Note that this config is false by default and operators should set it to true to disallow fingerprinting link local addresses.

Reply:
I was hesitant to write the default into this narrative lest it change, because it's documented in the disallow_link_local information.  What about:

> If no non-local IP addresses are found, Nomad could fingerprint link-local IPv6 addresses based on the "fingerprint.network.disallow_link_local" configuration value

Reply:
Sounds good.